python/lsst/ts/genericcamera/driver/simulatorcamera.py

```diff
@@ -461,6 +461,17 @@ async def readout(self):
         # Reset readout state
         self.readout_state = 0
 
+    def get_static_configuration_for_key_value_map(self) -> str:
```

Comment:
Coming back to this.. maybe this should be a static method.. I don't know.. there are some implications to that.. But if the idea is to be fully static, maybe make it so.

Reply:
Since it may require parts of the camera interface being up, it can't be a true static function. I'll fixup the name.